deploy/probers/postgres_write_contentnode_probe.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+import os
+from datetime import datetime
+
+import psycopg2
+from base import BaseProbe
+
+# Use dev options if no env set
+DB_HOST = os.getenv("DATA_DB_HOST") or "localhost"
+DB_PORT = 5432
+DB_NAME = os.getenv("DATA_DB_NAME") or "kolibri-studio"
+DB_USER = os.getenv("DATA_DB_USER") or "learningequality"
+DB_PASSWORD = os.getenv("DATA_DB_PASS") or "kolibri"
+TIMEOUT_SECONDS = 2
+
+
+class PostgresWriteContentnodeProbe(BaseProbe):
```

Comment:
Shouldn't this be marked develop_only? It is inserting a content node.

Reply:
Addressed in the latest commit!